crawler.js

```diff
@@ -381,7 +389,6 @@ class Crawler {
         console.warn(e);
       }
     });
-
```

Comment:
One place to add the page would be in the task() function above after the driver call:
```
this.createPage(opts.data.url)
```